decidim-conferences/spec/controllers/conferences_controller_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 module Decidim
   module Conferences
-    describe ConferencesController do
+    describe RegistrationTypesControllerController do
```

Comment:
```suggestion
    describe ConferencesController do
```

No need to change the described class